docs/backend-api/resources/tracking/tracker/readings.md

```diff
@@ -100,9 +121,10 @@ Gets last values for all metering sensors, state values and counters on multiple
 
 #### parameters
 
-| name     | description                                                                         | type | format          |
-|:---------|:------------------------------------------------------------------------------------|:-----|:----------------|
-| trackers | An array of tracker IDs (aka "object_id"). Trackers must belong to authorized user. | int  | [999199,991999] |
+| name        | description                                                                                        | type   | format          |
+|:------------|:---------------------------------------------------------------------------------------------------|:-------|:----------------|
+| trackers    | An array of tracker IDs (aka "object_id"). Trackers must belong to authorized user.                | int    | [999199,991999] |
+| sensor_type | Optional. If specified, state values and counters will be omitted. Used to filter sensors by type. | string | "fuel"          |
```

Comment:
Please note that this is an enum and add an internal link to the appropriate list.